scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -61,6 +62,21 @@ import java.util.SortedSet
 
 import kotlin.math.absoluteValue
 
+/**
+ * A wrapper for [ScanCode](https://github.com/nexB/scancode-toolkit).
+ *
+ * This scanner can be configured in [ScannerConfiguration.scanner] using the key "ScanCode". It offers the following
+ * configuration options:
+ *
+ * * **"commandLine"**: Command line options that modify the result. These are added to the [ScannerDetails] when
```

Comment:
Why is this bold (?) using `**`?

Reply:
Actually I wanted to make all names bold because it looks better, changed that.